great_expectations/execution_engine/execution_engine.py

```diff
@@ -468,7 +441,8 @@ def add_column_row_condition(
 
         Args:
             domain_kwargs: the domain kwargs to use as the base and to which to add the condition
-            column_name: if provided, use this name to add the condition; otherwise, will use "column" key from table_domain_kwargs
+            column_name: if provided, use this name to add the condition; otherwise, will use "column" key from
+            table_domain_kwargs filter_null: if true, add a filter for null values
```

Comment:
I think a newline got moved here?

Reply:
@billdirks Nice catch -- thank you (fixed).